prometheus-exporters/openstack-exporter/templates/openstack-exporter-openstack-config.yaml

```diff
@@ -17,6 +17,12 @@ data:
       cinderbackend:
         collector: openstack_exporter.collectors.cinderbackend.CinderBackendCollector
         enabled: True
+        {{- if .Values.collectors.cinderbackend.expected_sharding_backends }}
```

Comment:
This prohibits us to set this to an empty list. The default, that would be taken from the `openstack-exporter`, is also not an empty list.

Reply:
So, would you prefer getting rid of the if ?